go.mod

```diff
@@ -3,10 +3,13 @@ module github.com/ClearcodeHQ/magex
 go 1.16
 
 require (
```

Comment:
please run `go mod tidy`. I have a feeling that most of those packages are not needed.

Reply:
been there done that